MODULE.bazel

```diff
@@ -10,9 +10,22 @@ bazel_dep(name = "aspect_bazel_lib", version = "1.27.2")
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "rules_nodejs", version = "5.8.2")
 bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "gazelle", version = "0.28.0", repo_name = "bazel_gazelle")
+bazel_dep(name = "buildifier_prebuilt", version = "6.0.0.1", dev_dependency = True)
+bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.4.1", dev_dependency = True)
+
+ext = use_extension("@aspect_bazel_lib//lib:extensions.bzl", "ext")
+
+ext.host()
+
+use_repo(ext, "aspect_bazel_lib_host")
 
 node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
 
+node.toolchain(
+    node_version = "16.9.0"
+)
```

Comment:
WORKSPACE registers this specific version. If we don't do it here, one of our tests fails which checks the npm version.